configure.ac

```diff
@@ -581,9 +581,18 @@ AS_CASE([${RKT_ACI_ARCH}],
 RKT_IF_HAS_FLAVOR([${RKT_STAGE1_FLAVORS}], [coreos],
                   [AS_CASE([${RKT_ACI_ARCH}],
                            [aarch64],
-                                    [RKT_STAGE1_COREOS_INTERPRETER="/usr/lib/ld-linux-aarch64.so.1"],
+                                    [RKT_STAGE1_INTERPRETER="/usr/lib/ld-linux-aarch64.so.1"],
                            [amd64],
-                                    [RKT_STAGE1_COREOS_INTERPRETER="/usr/lib/ld-linux-x86-64.so.2"])])
+                                    [RKT_STAGE1_INTERPRETER="/usr/lib/ld-linux-x86-64.so.2"])])
+
+# Set the path to the interpreter binary for src flavor.
+RKT_IF_HAS_FLAVOR([${RKT_STAGE1_FLAVORS}], [src],
+                  [AS_CASE([${RKT_ACI_ARCH}],
+                           [aarch64],
+                                    [RKT_STAGE1_INTERPRETER="/lib64/ld-linux-aarch64.so.1"],
+                           [amd64],
+                                    [RKT_STAGE1_INTERPRETER="/lib64/ld-linux-x86-64.so.2"])])
+
```

Comment:
I don't think we really care about other archs with the src flavor, it's there just for testing purposes. I'll add "src" to the unsupported archs above.